returnn/frontend/_backend.py

```diff
@@ -1293,6 +1293,22 @@ def tensor_array_get_item(cls, tensor_array: TensorArrayType, index: Union[int,
             return tensor_array[index]
         raise NotImplementedError
 
+    @classmethod
+    def gradient_checkpoint(cls, fn: Callable[[Tensor], Tensor], *args: List[Tensor]) -> Tensor:
```

Comment:
Is this exactly like `recompute_grad`? I think i would rather then call it `recompute_grad`.